vllm/beam_search.py

```diff
@@ -13,6 +13,7 @@ class BeamSearchSequence:
     tokens: List[int]
     cum_logprob: float = 0.0
     text: Optional[str] = None
+    finish_reason: Optional[str] = None
```

Comment:
Is this necessary?

Reply:
Hi, thank you for the comment.
Yes I think so. Besides the finish_reason I also added the `stop_reason` in my latest commit to this class. These two values are passed to `CompletionOutput` when the generation is done